showcase-v1alpha2/src/v1alpha2/echo_client_config.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
The new API configuration doesn't have knobs for api-producers to alter this behavior. Can we remove this and put sensible defaults in gax?